### ISSUE TITLE
Will there be plans to include PGNs for the openings?

### ISSUE THREAD

**Issue by nayyala** [pwenker/chessli#13]
while also quizzing for the move sequence of an opening, will there be an addition to the flashcard and software to add a PGN to play it out visually?

**Comment by pwenker** [comment]
I usually try to play the opening in my head to improve visualization skills. Though it's probably still useful to have the board on the card.
A PGN is probably not needed, as the board and opening move sequence can be generated solely by the starting FEN (I can add it  in the card template) and the move sequence (already on the present as a field on the flashcard). 
So I guess I can just adapt the html/javascript on the `Chessli Opening` anki card template to make that work. I'll give it a try in the next days.

**Comment by nayyala** [comment]
that's a great point! and the lichess link i realized is sufficient.
If you choose to change it that's perfectly fine! I think i'll be able to make do with your current setup :D

**Comment by pwenker** [comment]
As an update, I succeeded adapting these card templates: https://ankiweb.net/shared/info/1082754005 to the `chessli` format. 
I already tested it, and it's really smooth and nice. Then we'll have an interactive chess board for the opening cards and all the other information including the lichess link to the game are still there. It will look something like this:

![demo](https://i.imgur.com/aDgeVvx.png)

However, the original creator is going to release some breaking changes to the template (possible this weekend), so I'll wait after that and then upload the new shared deck.
